### PR TITLE
removed smart sheets, reworked tagging

### PR DESCRIPTION
**Pull request by jsully1720**

### CODE REVIEW

File: python_examples/uuid_tagging/.gitignore [hunk 1]
```diff
@@ -116,3 +116,9 @@ venv.bak/
 # Token files
 .auth-token
 auth-token
+
+# raw token file 
+token_aa7123
+
+# poetry lock file
+poetry.lock
```
Comment (brian-l) [inline python_examples/uuid_tagging/.gitignore]: the poetry lock file should be committed

```suggestion
```

**Review by brian-l — CHANGES_REQUESTED** [review]

**Review by charles-d-burton — APPROVED** [review]
LGTM

**Comment by brian-l** [comment]
can you commit the poetry.lock you used?

**Review by brian-l — APPROVED** [review]

**Comment by bemcculley** [comment]
LGTM

**Pull request merged**